GliaWidgets/Sources/View/Call/CallStyle.RemoteConfig.swift

```diff
@@ -0,0 +1,144 @@
+import UIKit
+
+extension CallStyle {
+    /// Apply call configuration from remote configuration
```

Comment:
To make it more consistent with other texts:
```suggestion
    /// Applies call configuration from remote configuration.
```